pkg/agent/testdata/AKSUbuntu1604+EnableHostsConfigAgent/CustomData

```diff
@@ -0,0 +1,174 @@
+
+
+
+
+
+
+
+
```

Comment:
can you do me a favor and see if we can get rid of these empty lines? {{ - should help

Reply:
doesn't help with these empty lines. So I'll keep the style align with other places.

Reply:
o, you're right. It can be removed. Should make generate first and then generate the test data.